docs/developers/v2/hacking-with-yearn.md

```diff
@@ -2,145 +2,153 @@
 
 This page is for the hackers and buidlers who are currently building, or planning to build, something cool with Yearn Finance. This guide details all the info you need to start hacking!
 
-# Core Tools
+## Core Tools
 
 - [Solidity](https://docs.soliditylang.org/en/v0.8.6/index.html)
 - [Brownie](https://eth-brownie.readthedocs.io/en/stable/)
 - [ethers.js](https://docs.ethers.io/v5/)
 - [Web3.js](https://web3js.readthedocs.io/en/v1.4.0/)
-- [Ganache](https://trufflesuite.com/ganache/) 
+- [Ganache](https://trufflesuite.com/ganache/)
 - [Foundry](https://github.com/nicolasgarcia214/damn-vulnerable-defi-foundry)
```

Comment:
Is there a reason this doesn't link to the main foundry github repo? https://github.com/foundry-rs/foundry

Reply:
yeah change the link, nice catch